### PR TITLE
Prepare releases 1.3.11 and 1.4.2

### PR DESCRIPTION
**Pull request by ignasi35**
* Prepares `changelog*.md` files.
* Updates broken links from past blog posts to appropriate `changelog.md`
* Drafts blog ANN

### CODE REVIEW

File: src/blog/lagom-1-2.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The Lagom team is proud to announce the release of Lagom 1.2!  This release intr
 
 If you've been following along with the RCs, there have only been minor changes since RC2, but if you're upgrading from 1.1.0 or earlier, please be sure to follow the [migration guide](/documentation/1.2.x/java/Migration12.html), as there are some changes you'll need to make to your project.
 
-Here's a recap of the major new features that were announced with the release candidates. You can see more details in the [change log](/changelog.html) or on [GitHub](https://github.com/lagom/lagom/issues?utf8=%E2%9C%93&q=milestone%3A1.2.0).
+Here's a recap of the major new features that were announced with the release candidates. You can see more details in the [change log](/changelog-archive.html) or on [GitHub](https://github.com/lagom/lagom/issues?utf8=%E2%9C%93&q=milestone%3A1.2.0).
```
Comment (ignasi35) [inline src/blog/lagom-1-2.md]: Lots of broken links in past blog posts when we moved `changelog.md` to separate files.

File: src/blog/lagom-1-4-2-and-1-3-11.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+Special thanks to the following contributors who helped with the release 1.4.2:
+
+```
+commits  added  removed
+      9     291      56 Ignasi Marimon-Clos
+      6     176     139 Tim Moore
+      1      12       0 Jason Pickens
+```
```
Comment (ignasi35) [inline src/blog/lagom-1-4-2-and-1-3-11.md]: This is pending completion of https://github.com/lagom/lagom/milestone/28

File: src/main/markdown/changelog.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+* [1277](https://github.com/lagom/lagom/issues/1277) Adds Kafka message timestamps to Metadata (Ignasi Marimon-Clos)
+* [1275](https://github.com/lagom/lagom/issues/1275) Document required behaviour of flow to message broker subscriber (Jason Pickens)
+* [1265](https://github.com/lagom/lagom/issues/1265) Build Maven plugin and support for Scala 2.12 only (Tim Moore)
+* [1263](https://github.com/lagom/lagom/issues/1263) Fix Cassandra and JDBC conflicts in TestKit (Tim Moore)
+* [1259](https://github.com/lagom/lagom/issues/1259) Adds more akka-misc serializers on the Migration guides and Mvn archetype (Ignasi Marimon-Clos)
+* [1258](https://github.com/lagom/lagom/issues/1258) Remove unnecessary workarounds in ServiceTestSpec (Tim Moore)
+* [1257](https://github.com/lagom/lagom/issues/1257) Applies suggested solution: use Seq(EmptyTree) for empty traits (Ignasi Marimon-Clos)
+* [1246](https://github.com/lagom/lagom/issues/1246) Removes docs wrt the metrics service and its endpoints (Ignasi Marimon-Clos)
+
```
Comment (ignasi35) [inline src/main/markdown/changelog.md]: I preferred putting stuff into `1.3.x` rather than here.

Comment (renatocaval) [reply src/main/markdown/changelog.md]: IMO, we should stop producing this changelog files. We have this information on GitHub already and we are providing the link in the announcement.
I also doubt that someone is really reading the changelog.

**Pull request merged**